### DISCUSSION TITLE
Please put the right click context menu item back

### DISCUSSION THREAD

**Discussion by MartinR577** [aclap-dev/video-downloadhelper#575] [Help and Q&A]
### Your question

before the update there was a right click context menu option for Video-downloadhelper. it would give you options like which resolution to save.  please add that feature back.

### Addon & CoApp version and details

this copy details option is no longer available in this new version

**Comment by paulrouget** [comment]
It should be fixed in upcoming version: v9.0.1.18.

**Comment by paulrouget** [comment]
Which browser and what is the version of your browser?

**Reply by MartinR577** [reply]
Chrome Version 125.0.6422.113 (Official Build) (64-bit)and Brave [Version 1.66.115 Chromium: 125.0.6422.112 (Official Build) (64-bit)](https://brave.com/latest/)

**Reply by paulrouget** [reply]
To be clear, are you talking about the right click on the page? Or right click on the addon popup?

This:

<img width="481" alt="Screenshot 2024-05-25 at 23 26 17" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/8d28f686-c5d4-417d-93f4-d997dd0290cd">

Or maybe, near the "Download" button, there's 3 dots.

<img width="335" alt="Screenshot 2024-05-25 at 23 27 08" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/a50a5fde-80ff-4d5a-814f-b2f5c2190b7a">

**Reply by MartinR577** [reply]
yes, previously when you right clicked on a video you got the video download helper menu item.  and you could choose the resolution for the video you right clicked on.   as you show in your first image.

**Reply by paulrouget** [reply]
@MartinR577 you should still see this:

1. make sure you're using the latest version of the addon
2. do this, and copy/paste to the logs here: https://github.com/aclap-dev/video-downloadhelper/wiki/How-to-enable-logs

**Reply by GrampaWildWilly** [reply]
I'm on Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.  I'm getting this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/c88f4972-065b-4792-818d-0f102e4fb741)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/302d4a11-beb9-4eea-9e47-101306ca9b3c)

Give me few moments while I mess with that logs thing.

**Reply by GrampaWildWilly** [reply]
First, I must STRENUOUSLY complain that those logs contain both your E-mail address, which I suppose is only annoying, as well as your VDH license number.  YOUR LICENSE NUMBER!!!!!!!  You've got to be kidding.  Those things are utterly irrelevant to problem diagnosis.  And shame on you for actually making this include the license number.  The most you could possibly need is a simple indication that I have a license, not the actual license itself.

Anyway, here is what I did:

1. Opened the YouTube home page.
2. Set up the logging thing.
3. Popped up the context menu on something on the YouTube page, like I show in one of my images above.
4. Captured the attached log.

To my uneducated eye, it is singularly uninformative.  But maybe it says something to you.

[vdh-logs.txt](https://github.com/aclap-dev/video-downloadhelper/files/15445865/vdh-logs.txt)

**Reply by MartinR577** [reply]
it shows the page i'm on but i don't see where that add a right menu item wheen you click on a video. this is windows 10. I found the option  to use old ui and the right context menu is still not there
[vdh-logs.txt](https://github.com/aclap-dev/video-downloadhelper/files/15449252/vdh-logs.txt)

**Reply by GrampaWildWilly** [reply]
You published your VDH license in public, @MartinR577.  I warned people about that.  Your E-mail address is on line 16 of what you posted & your VDH license is on line 17.  You need to edit your post as soon as you can & remove that attachment.  Then you need to edit the log file & remove the offending strings.  Then you can post the log file again.

See, @paulrouget Paul?  I told you this was a terrible thing that you've done.  You need to fix it right away.

**Reply by paulrouget** [reply]
@GrampaWildWilly eeeerg. thanks!!

**Reply by GrampaWildWilly** [reply]
Oh good.  I yelled at you in another thread before I read this one.  I'm glad you're on it.

**Comment by jeff15110168** [comment]
version 9 now doesn't show the size of the file to be downloaded when right clicking. Prior versions had this. can this please be re-added?

**Reply by paulrouget** [reply]
Can you give me an URL where it doesn't show the size?

**Comment by MartinR577** [comment]
I see now the change in behavior. Before i would right click  directly on a playing video and get the context menu. Now i get it if i right click somewhere else on the page of the video but not directly on the video. in my case  this works as many web sites replace the stardard right click menu  or disable right click completely over the video .
